app/javascript/src/components/Contacts/Create/index.jsx

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+
+import { Check } from "@bigbinary/neeto-icons";
+import { Pane, Typography, Button } from "@bigbinary/neetoui/v2";
+import { Input, Select } from "@bigbinary/neetoui/v2/formik";
+import { Formik, Form } from "formik";
+import PropTypes from "prop-types";
+
+import formInitialValue from "constants/formInitialValues";
+import formValidationSchema from "constants/formValidationSchemas";
+
+import { ROLE_OPTIONS } from "../constants";
+
+const CreateContact = ({ showPane, setShowPane, onSubmit }) => {
+  return (
+    <Pane onClose={() => setShowPane(false)} isOpen={showPane}>
+      <Formik
+        initialValues={formInitialValue.createContactForm}
+        onSubmit={onSubmit}
+        validateOnChange={false}
+        validateOnBlur={false}
+        validationSchema={formValidationSchema.createContactForm}
+      >
+        {({ isSubmitting }) => (
+          <Form>
+            <Pane.Header>
+              <Typography style="h2" weight="semibold">
+                Add New Contact
+              </Typography>
+            </Pane.Header>
+
+            <Pane.Body>
+              <div className="w-full flex flex-col ">
+                <div className="flex flex-row w-full">
+                  <Input
+                    name="firstName"
+                    type="text"
+                    label="First Name"
+                    placeholder="Enter first name"
+                    className="my-4 pr-2"
```

Comment:
We shouldn't be using padding here. 
Please use `space-x-*` utils on the parent wrapper.